README.md

```diff
@@ -1,4 +1,4 @@
-[![Build Status](https://travis-ci.org/snudler6/time-travel.svg?branch=master)](https://travis-ci.org/snudler6/time-travel)
+[![Build Status](https://travis-ci.org/snudler6/time-travel.svg?branch=master)](https://travis-ci.org/snudler6/time-travel) [![Build status](https://ci.appveyor.com/api/projects/status/y13ewnvmj0muoapf/branch/master?svg=true)](https://ci.appveyor.com/project/snudler6/time-travel/branch/master)
 
 # time-travel
 python time libraries mocking
```

Comment:
Update the tests shown in the doc to use non-zero timestamp.

Reply:
Done